build.sh

```diff
@@ -199,16 +209,39 @@ build_combined() {
         -framework "$os_path" "${simulator_framework[@]}"
 }
 
+change_schema_configuration() {
+    local scheme="$1"
+    filename="Realm.xcodeproj/xcshareddata/xcschemes/${scheme}.xcscheme"
+    sed -i '' "s/buildConfiguration = \"Debug\"/buildConfiguration = \"$CONFIGURATION\"/" "$filename"
+}
+
+set_configuration_for_distribution() {
+    filename="Configuration/RealmSwift/RealmSwift.xcconfig"
+    sed -i '' "s/REALM_BUILD_LIBRARY_FOR_DISTRIBUTION = NO;/REALM_BUILD_LIBRARY_FOR_DISTRIBUTION = YES;/" "$filename"
+}
+
 clean_retrieve() {
-  mkdir -p "$2"
-  rm -rf "$2/$3"
-  cp -R "$1" "$2"
+    mkdir -p "$2"
+    rm -rf "$2/$3"
+    cp -R "$1" "$2"
 }
 
 plist_get() {
     /usr/libexec/PlistBuddy -c "Print :$2" "$1" 2> /dev/null
 }
 
+update_core_revision() {
+    version="$(sed -n 's/^REALM_CORE_VERSION=\(.*\)$/\1/p' "${source_root}/dependencies.list")"
+
+    git clone https://github.com/realm/realm-core.git
+    cd realm-core
+    git checkout v$version
+    last_commit=$(git rev-parse HEAD);
+    cd ..
+    sed -i '' "s/\"revision\" : .*,/\"revision\" : \"$last_commit\",/" examples/installation/SwiftPackageManagerDynamic.notxcodeproj/project.notxcworkspace/xcshareddata/swiftpm/Package.resolved
+    rm -rf realm-core
+}
```

Comment:
Building the SPM tests with `xcodebuild -resolvePackageDependencies -IDEPackageOnlyUseVersionsFromResolvedFile=NO -IDEDisableAutomaticPackageResolution=NO` makes this unnecessary.